README.md

```diff
@@ -80,22 +80,48 @@ To set your realm, use a subdomain, such as ingest.us1.signalfx.com or ingest.eu
 
 ## Step 3: Set environment variables
 
-1. Set SIGNALFX_AUTH_TOKEN with your correct access token. Review the following example. 
+1. Set SIGNALFX_ACCESS_TOKEN with your correct access token. Review the following example. 
     ```bash
-     SIGNALFX_AUTH_TOKEN=signalfx token
-    ```
-2. Set the ingest endpoint URL with the domain that corresponds to your realm. Review the following example.   
-    ```bash
-    SIGNALFX_INGEST_ENDPOINT=[https://pops.signalfx.com]
+     SIGNALFX_ACCESS_TOKEN=access-token
     ```
+
+2. If you use Smart Gateway, or want to ingest directly from a realm other than us0, then you must set at least one endpoint variable. (For environment variables, SignalFx defaults to the us0 realm. As a result, if you are not in the us0 realm, you may need to set your environment variables.) There are two options:
```

Comment:
under the impression we don't want to call out PG details unless they are necessary, which I'm not sure is the case here.  Collector instead or drop it?

Reply:
it looks like we can just substitute "Smart Gateway" with "OpenTelemetry Collector" - is that true?

Reply:
Updated. Replaced mentions of gateway with otel collector and updated the endpoint URLs to work with Otel's zipkin receiver.